src/utils/push-event-utils.ts

```diff
@@ -43,3 +44,8 @@ export const isEventForTrackingBranch = (event: PushEvent, trackingBranch: strin
 
   return event.ref === trackingBranchRef;
 };
+
+export const isUpdateCompassComponentDataManager = (
```

Comment:
If this utility only needs to check whether this contains a `lastSyncEvent`, a `has` prefix in the method name could clarify.

`hasUpdateCompassComponentDataManagerLastSyncEvent` could be a mouthful. Maybe Maybe `hasLastSyncEvent`? The typed parameter should clarify what this applies to.

Reply:
Good catch! Thank you @eserranovalle